fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -251,8 +251,7 @@ public synchronized AddResult addTablet(TabletSchedCtx tablet, boolean force) {
         // if this is not a BALANCE task, and not a force add,
```

Comment:
Please update and add some explain to the comments.

Reply:
done.  we should also limit the num of adding balance task,  because we don't want the sched queue to become too large.